docs/filters.md

```diff
@@ -148,7 +148,9 @@ Matches all issues with a body (or description) containing the given string.
 
 *Expects a string*
 
-Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `curr-2`, `curr-1`, `curr+1`, `curr+2`, etc. (no space before or after `-`/`+`) can then be used to refer to milestones before or after `current`, sorted by due date.
+Matches all issues associated with any milestones whose names contain the given string.
+
+`current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `curr-2`, `curr-1`, `curr+1`, `curr+2`, etc. (no space before or after `-`/`+`) can then be used to refer to milestones before or after `current`, sorted by due date. Milestones with no due date will not be considered in resolving aliases. If there is no open milestone, `current` will refer to no milestone, while `current-1` will refer to the last closed milestone.
```

Comment:
Does the reader know what 'alias' refer to at that point? i..e Has the term being defined somewhere just above? Minimizing terms makes the text easier to understand.